# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/nsICanvasRenderingContextInternal.h
# Commit: 047bcbf3769f
# Full Hash: 047bcbf3769f8dcdeda5f00e8dd1b0cad893e602
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2021-12-10 21:58:52
# Regressor Bug: 1736177
# File Overlap Count: 2
# Description:
#   Bug 1736177 - Part 8. Remove unused visibility observer from HTMLCanvasElement. r=jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130784
# ==============================================================================

diff -r 2859a91f3a73 -r 047bcbf3769f dom/canvas/nsICanvasRenderingContextInternal.h
--- a/dom/canvas/nsICanvasRenderingContextInternal.h	Fri Dec 10 02:57:54 2021 +0000
+++ b/dom/canvas/nsICanvasRenderingContextInternal.h	Fri Dec 10 02:57:54 2021 +0000
@@ -183,8 +183,6 @@
     return nsString();
   }
 
-  virtual void OnVisibilityChange() {}
-
   virtual void OnMemoryPressure() {}
 
   virtual void OnBeforePaintTransaction() {}